packages/edit-navigation/src/components/header/style.scss

```diff
@@ -42,6 +43,15 @@
 			display: none;
 		}
 	}
+
+	> .edit-navigation-header-inserter-toggle.has-icon.has-icon.has-icon {
```

Comment:
We need a better way to override default toolbar styles.

Reply:
How do other editors handle this issue?

Reply:
We have similar in the Widget Editor, so not sure what's the best option is here.

Reply:
Probably fine then!